proposals/0304-structured-concurrency.md

```diff
@@ -935,13 +1071,33 @@ All of the changes described in this document are additive to the language and a
 
 ## Revision history
 
+### Review changes
+
+Changes after first review:
+
+* `Task.current` now returns an optional `Task`: `var current: Task? { get }`, which depending on context it is called from might be `nil`.
+  * This API is not intended to be used "a lot", and if sure a task will be available one can always force unwrap it.
+  * Most usages of tasks are rather intended to go through the static functions/properties on Task which implicitly works on the current task.
+* `Task.unsafeCurrent` becomes a top-level `withUnsafeCurrentTask { maybeUnsafeTask in }`
+  * This better explains the intended semantics of not escaping storing the unsafe task reference.
+* Adopt `spawn` terminology for "spawning tasks" 
+  * `spawnDetached` becomes `spawnDetached`
+  * `Task.Group`'s `group.add` becomes `group.spawn`
+  * Creating a child task will eventually be `spawn <something>`
```

Comment:
I don't think we should bullet point changes that haven't been made or decided yet.

Reply:
Okey